### PR TITLE
Update Templates

### LINKED ISSUE #124: update templates
Copy and suitably modify the following templates.
 
1. PR and Issue templates from [SCAT](https://github.com/DeveloperCAP/SCAT/tree/master/.github).
1. Code of conduct from [autolabcli](https://github.com/AutolabJS/autolabcli/blob/master/CODE_OF_CONDUCT.md)
1. Contributing guidelines from [SCAT](https://github.com/DeveloperCAP/SCAT/blob/master/CONTRIBUTING.md)

All the necessary wiki pages need to be created on MLCAT repository and linked added to the local wiki pages.

### PR DESCRIPTION
**Pull request by naveen-hr**
## What? Why?
Fix #124  

Changes proposed in this pull request:
- CODE_OF CONDUCT.md added
- CONTRIBUTING.md added
- ISSUE_TEMPLATE.md added
- PULL_REQUEST_TEMPLATE.md added
- Above files added to .github/ 

## Checks
- [x] Single commit and [No merge commits](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/)
- [x] Make sure you have added required documentation
- [ ] If you have refactored a code, make sure that you have compared the outputs pre and post refactoring. Add both pre and post refaoring screenshots below if the output is visual

## Notify reviewers
@prasadtalasila

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# How to contribute
+
+Thank you for contributing to Slack Community Analysis Tool (SCAT)!
```
Comment (prasadtalasila) [inline .github/CONTRIBUTING.md]: The tool name should be Mailing List Community Analysis Tool (MLCAT).

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+> "The fork & pull model lets anyone fork an existing repository and push changes to their personal fork without requiring access be granted to the source repository. The changes must then be pulled into the source repository by the project maintainer. This model reduces the amount of friction for new contributors and is popular with open source projects because it allows people to work independently without upfront coordination."
+
+## Branches
+MLCAT currently maintains two branche(s):
+* **master**:  this is the stable branch to be used for installation.
+* **development**: all of the development happens on dev and is periodically merged with master
+ 
+## Code style guidelines
+The [style guidelines](https://github.com/DeveloperCAP/SCAT/wiki/Coding-Guidelines) wiki page defines the source file coding standards used in this project.
```
Comment (prasadtalasila) [inline .github/CONTRIBUTING.md]: Please update the URL.

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+The following case convention is used in this project: module_name, package_name, ClassName, method_name, ExceptionName, function_name, GLOBAL_CONSTANT_NAME, global_var_name, instance_var_name, function_parameter_name, local_var_name. 
+
+## Submitting pull requests 
+* Always check out the `development` branch and submit pull requests against it. Backports to prior versions will be considered on a case-by-case basis and reflected as the fix version in the issue tracker.
+* Maintain a clean commit history even during review. That means that commits are rebased and squashed if necessary, so that each commit clearly contains one change and there are no extraneous fix-ups.
+* The pull request should be motivated, i.e. what does it fix, why, and if relevant how.
+* If possible, include an example in the description, that could help all readers (including project members) to get a better picture of the change.
+* Avoid other runtime dependencies unless abslutely necessary 
+* Ensure that your pull request follows the [style guidelines](https://github.com/DeveloperCAP/SCAT/wiki/Coding-Guidelines)
```
Comment (prasadtalasila) [inline .github/CONTRIBUTING.md]: Please update the URL.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+## Checks
+ - [ ] Squash your commits down to one, or a few, discreet changesets before submitting a pull request
+ - [ ] Read the [contributing guide](https://github.com/DeveloperCAP/MLCAT/blob/development/.github/CONTRIBUTING.md)
+ - [ ] PR should be motivated, i.e. what does it fix, why, and if relevant how
+ - [ ] If possible, include an example in the description, that could help all readers (including project members) to get a better picture of the change
+ - [ ] Avoid other runtime dependencies unless abslutely necessary 
+ - [ ] Rebase your commit history so that each commit is meaningful and states the intention behind the change 
+ - [ ] Ensure that your pull request follows the [style guidelines](https://github.com/rohangoel96/Slack-Community-Analysis-Tool/wiki/Coding-Guidelines)
```
Comment (prasadtalasila) [inline .github/PULL_REQUEST_TEMPLATE.md]: please update the URL.

**Review by prasadtalasila — CHANGES_REQUESTED** [review]
@naveen-hr please make the requested changes.

**Comment by naveen-hr** [comment]
The requested changes have been done.

**Review by prasadtalasila — APPROVED** [review]

**Pull request merged**